.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,75 @@
+name: "Found a Bug! 🐛"
+title: "🐛 [BUG] - <title>"
+description: "Use this template to report a bug. Thank you!"
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks in advance for taking the time to file this issue!
+        We hope this simple template will improve issue resolution time and minimize the need for _context discovery_ discussions.
+        ______
+        **Note**: We encourage you to [submit an RFC](https://github.com/neon-bindings/rfcs) if you would like to propose a **design change** or a **new feature** for the Neon API or the Neon command-line tool.
+        ______
+
+        In addition to the `bug` label, consider adding others you might find relevant:
```

Comment:
This table is nice documentation for us to have somewhere, but I don't think it belongs in our issue template. I don't think we have a way to enable non-maintainers to attach labels, and I think the ROI isn't there anyway (the benefit of getting help with triage doesn't pay for the friction it adds to casually filing issues).

Reply:
Good feedback, thanks.  I was struggling to maintain that same ratio, ultimately I figured it's easier to cut things out than not have enough detail.   Maybe the labels table is best moved to the contributing doc?